tools/cldr-code/src/main/java/org/unicode/cldr/util/CharacterFallbacks.java

```diff
@@ -25,7 +25,7 @@ private CharacterFallbacks() {
                 DtdData.getInstance(DtdType.supplementalData).getDtdComparator(null);
 
         for (Iterator<String> it =
-                        characterFallbacks.iterator("//supplementalData/characters/", comp);
+                        characterFallbacks.iteratorDefault("//supplementalData/characters/", comp);
                 it.hasNext(); ) {
             String path = it.next();
```

Comment:
All the callers of `iterator` outside of CLDRFile have become `iteratorDefault`. Note, BTW, that this loop, and many like it, should be simplified by calling `iterableDefault` instead of `iteratorDefault`. I've postponed that simplification, pending decision on the overall approach. Also, including the word "default" in the method names might be temporary; for now, it helps to indicate which code is affected by changing the default behavior from excluding to including extra paths